pint_server/pint_db_migrate/versions/b720d8b8b775_set_server_type_name_not_null.py

```diff
@@ -0,0 +1,62 @@
+"""Set server type, name NOT NULL
+
+Revision ID: b720d8b8b775
+Revises: b381843009e0
+Create Date: 2021-09-06 17:31:28.809521
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'b720d8b8b775'
+down_revision = 'b381843009e0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
We could potentially clean this up a bit by defining the ENUM once instead of using duplicate definitions here in the op.alter_column() statements.

The auto-generated code is a little dumb in this regard. ;-)

I'm sure that under the hood SQLAlchemy/PostgreSQL are detecting equivalency but it would probably make the code look cleaner.

Reply:
Yes, I know what you mean, and I have done some refactoring in other PRs but, as a rule thumb, if the auto-generated code by alembic does not need any change, leave like that (i.e. IPv6 PR)